app/models/provider.rb

```diff
@@ -140,7 +140,7 @@ def current_accredited_courses
 
   validates :provider_type, presence: true
 
-  validates :telephone, phone: { message: "^Enter a valid telephone number" }, if: :telephone_changed?
+  validates :telephone, phone: { message: "Enter a valid telephone number" }, if: :telephone_changed?
```

Comment:
Does this have any side effects in Publish? The carat is a convention introduced by https://github.com/DFE-Digital/teacher-training-api/blob/master/Gemfile#L82, and I wonder if this changes the error messages Publish receives?

Reply:
To clarify - this is relevant because Old Publish will continue serving contact-editing functionality until we start redirecting to New Publish.

Reply:
i've just added messages unique to the form, undid the original changes of removing the carats